src/stp/new_roles/Attacker.cpp

```diff
@@ -0,0 +1,18 @@
+//
+// Created by jordi on 17-03-20.
+//
+
+#include "stp/new_roles/Attacker.h"
+#include "stp/new_tactics/GetBall.h"
+#include "stp/new_tactics/DriveWithBall.h"
+#include "stp/new_tactics/KickAtPos.h"
+
+namespace rtt::ai::stp::role {
+
+Attacker::Attacker(std::string name) : Role(std::move(name)) {
+    // create state machine and initializes the first state
+    robotTactics = collections::state_machine<Tactic, Status, StpInfo>{tactic::GetBall()/*, tactic::DriveWithBall()*/, tactic::KickAtPos()};
```

Comment:
Why is DriveWithBall commented out?

Reply:
We might want to add that tactic so the attacker can position itself a bit better before shooting